packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart

```diff
@@ -81,22 +88,26 @@ abstract class AuthenticatorPage {
   }
 
   // Then I am signed in
-  void expectAuthenticated() {
-    expect(signOutButton, findsOneWidget);
+  Future<void> expectAuthenticated() async {
+    final inheritedBloc =
+        tester.widget<InheritedAuthBloc>(find.byKey(keyInheritedAuthBloc));
+    if (inheritedBloc.authBloc.currentState is! Authenticated) {
+      await nextBlocEvent(tester);
+    }
+    expect(inheritedBloc.authBloc.currentState, isA<Authenticated>());
   }
 
   /// Then I see User not found banner
-  Future<void> expectUserNotFound() async {
-    expectError('User does not exist.');
-  }
+  Future<void> expectUserNotFound() async => expectError(
+        Platform.isAndroid ? 'User not found' : 'User does not exist',
```

Comment:
Not sure why this discrepancy exists

Reply:
Ran into this on another test. Amplify-Android defines the messages in the library. I believe iOS is using the message from Cognito.